spec/peeps_seeds.sql

```diff
@@ -0,0 +1,6 @@
+TRUNCATE TABLE peeps RESTART IDENTITY;
+
+INSERT INTO peeps ("username", "comment", "time") VALUES
+('user_01', 'Hello world', '2022/04/12 00:14:00'),
+('user_01', 'This is awesome!', '2022/04/12 00:14:45'),
+('user_01', 'Bored.', '2022/04/12 00:56:57');
```

Comment:
Next time you can use multiple usernames as examples as it will give you more scope to test out ideas later!